examples/basic/main.tf

```diff
@@ -114,4 +114,5 @@ module "observability_agents" {
   logdna_add_cluster_name   = true
   # example of how to include / exclude metrics - more info https://cloud.ibm.com/docs/monitoring?topic=monitoring-change_kube_agent#change_kube_agent_log_metrics
   sysdig_metrics_filter = [{ type = "exclude", name = "metricA.*" }, { type = "include", name = "metricB.*" }]
+  sysdig_agent_tags     = var.sysdig_agent_tags
```

Comment:
update this to `sysdig_agent_tags = var. resource_tags`

can you also update this too:
`logdna_agent_tags = var.resource_tags`